.circleci/packages/influxdb3/control/post-install

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+BIN_DIR=/usr/bin
+DATA_DIR=/var/lib/influxdb3
+LOG_DIR=/var/log/influxdb3
+
+# Distribution-specific logic
+if [[ -f /etc/debian_version ]]; then
+    # Ownership for RH-based platforms is set in build.py via the `rmp-attr` option.
+    # We perform ownership change only for Debian-based systems.
+    # Moving these lines out of this if statement would make `rmp -V` fail after installation.
```

Comment:
AIUI, this is saying that on Debian-based systems, unconditionally chown and chmod the LOG_DIR and DATA_DIR. Does this happen on upgrades or just new installs? Why is this needed if post-install is already creating the directories correctly. This feels like something cargo-culted in from other packaging that was meant to fix a permissions problem for the other package that doesn't apply here, but maybe I'm missing something?

Reply:
This is a straight copy from oss 2.x so maybe @bnpfeife would know more?